asv_bench/asv.conf.json

```diff
@@ -43,13 +43,14 @@
     "matrix": {
         "numpy": [],
         "Cython": ["0.29.32"],
-        "matplotlib": [],
+        "pip+matplotlib>": ["3.6.1"],
```

Comment:
couldn't get this to resolve with conda https://github.com/pandas-dev/pandas/actions/runs/3518044244/jobs/5896518481